README.md

```diff
@@ -15,19 +15,21 @@ Simple shortened example for listing all pipelines:
 ```go
 import (
     "github.com/buildkite/go-buildkite/v3/buildkite"
+    "gopkg.in/alecthomas/kingpin.v2"
 )
-...
 
-config, err := buildkite.NewTokenConfig(*apiToken, false)
+var (
```

Comment:
Did we add this in for a reason? I don't see them in the original README. This isn't a requested change

Reply:
@mcncl more to have the READMEs' example in-line with our actual examples in that dir, though more the very shorthand example. We were also referencing `*org` and `*apiToken` without their actual assignment before so its more a concrete example just in case it was copy-pasted 👍

Reply:
Yeah, I think the `...` is intended to be a _other code here_ type thing.

Reply:
Oh yeah thats fair - I guess now its more concrete than anything :)